package.json

```diff
@@ -18,6 +18,11 @@
     "debug": "node-inspector & electron --debug=5858 . & xdg-open http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858",
     "doc": "jsdoc -c .jsdocrc",
     "lint": "jshint . --verbose --exclude-path=.jshintignore",
+    "postinstall": "npm run sia-repo && npm run sia-import",
+    "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
+    "sia-import": "(test -d app/dependencies/Sia || mkdir -p app/dependencies/Sia) && cp $GOPATH/bin/siad app/dependencies/Sia",
```

Comment:
I don't know why we need this rule - it hasn't been needed for installation for Luke + I. But maybe that's because we haven't done any clean installs since the rule got removed.